frontend/src/metabase/visualizations/echarts/graph/sankey/option/index.ts

```diff
@@ -67,7 +67,7 @@ export const getSankeyChartOption = (
       color: renderingContext.getColor("text-dark"),
       fontSize: SANKEY_CHART_STYLE.edgeLabels.size,
       textBorderWidth: SANKEY_CHART_STYLE.edgeLabels.textBorderWidth,
-      textBorderColor: renderingContext.getColor("white"),
+      textBorderColor: renderingContext.getColor("bg-white"),
```

Comment:
Using `var(--mb-color-background)` would break static viz, so we use `bg-white` instead.